Formula/terraform.rb

```diff
@@ -101,6 +101,9 @@ def install
         count = 4
       }
     EOS
-    system "#{bin}/terraform", "graph", testpath
+    cd testpath do
```

Comment:
I wasn't sure because it was previously passed as a parameter, but I just tested it and it works without the cd. Pushing new commit.